plugins/home/api-report.md

```diff
@@ -39,6 +39,13 @@ export const ComponentAccordion: ({
   ContextProvider?: ((props: any) => JSX.Element) | undefined;
 }) => JSX.Element;
 
+// Warning: (ae-missing-release-tag) "ComponentRenderer" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
perhaps add a @public tag for this one? :)

Reply:
added! 

I have to go through this entire api report at some point, but as this PR is not really about home I'll do it separately.